openmc/statepoint.py

```diff
@@ -389,7 +389,7 @@ def tallies(self):
                         new_filter.mesh = self.meshes[key]
 
                     # Add Filter to the Tally
-                    tally.add_filter(new_filter)
+                    tally.filters.append(new_filter)
```

Comment:
This is a very good point. I had wrongly assumed that calling append would effectively trigger the property setter, but it seems that it doesn't. Thinking about it again now, it makes sense that it wouldn't. In theory, we could subclass list and override the append method, forcing it to typecheck its elements and then have the setter always convert into the list subclass, but that seems a little overkill. Hmm, let me think more about this.